algorithms/aflplusplus/aflplusplus_mutation_hierarflow_routines.cpp

```diff
@@ -39,10 +39,11 @@ AFLMutCalleeRef<AFLplusplusState> HavocTemplate<AFLplusplusState>::operator()(
 
   using afl::dictionary::AFLDictData;
 
-  if (this->DoHavoc(mutator, *state.mutop_optimizer,
-                    aflplusplus::havoc::AFLplusplusCustomCases, "more_havoc",
-                    "more_havoc", state.orig_perf, stage_max_multiplier,
-                    option::STAGE_HAVOC)) {
+  if (this->DoHavoc(
+          mutator, *state.havoc_optimizer,
+          aflplusplus::havoc::AFLplusplusCustomCases<AFLplusplusState>(state),
```

Comment:
Well, this is kind of out of a whim. So I'll make them consistent.
But I'm going to add `<RezzufState>` to the rezzuf implementation rather than remove `<AFLplusplusState>` from this.

Reply:
Fixed in https://github.com/fuzzuf/fuzzuf/pull/92/commits/a92f25d5aa72cc3fa15837b0942b59bc85033570